examples/platformsh-kitchensink/README.md

```diff
@@ -237,6 +237,25 @@ lando ssh -c "curl localhost/solr.php" | grep "Result" | grep "OK"
 # Should run php mongodb commands successfully
 cd sink/php
 lando ssh -c "curl localhost/mongodb.php" | grep "Result" | grep "OK"
+
+# Should have OS Pid when running rabbitmqctl status
+lando ssh -u root -s rabbitmq -c "rabbitmqctl status" | egrep "OS PID: [0-9]+"
+
+# Should load rabbitmq management page
+cd sink/php
+lando ssh -s rabbitmq -c "curl localhost:15672" | grep "RabbitMQ Management"
+
+# Should run php rabbitmq php commands successfully
+cd sink/php
+lando ssh -c "curl localhost/rabbitmq.php" | grep "Result" | grep "OK"
+
+# Should show influxdb process running as the app user
+cd sink/php
+lando ssh -u root -s influxdb -c "ps aux|grep influxdb" | grep "^app"
+
+# Should show kafka process running
+# Temporarily commented out until kafka issues are resolved
+# lando ssh -u root -s kafka -c "ps aux|grep '\srunsv kafka'"
 ```
```

Comment:
@mikemilano you might need to throw in a dummy command here like 

```md
# Should show kafka process running
# Temporarily commented out until kafka issues are resolved
# lando ssh -u root -s kafka -c "ps aux|grep '\srunsv kafka'"
true
```

im not sure if the test framework knows how to parse a block that is all comments correctly